### ISSUE TITLE
[feature] limit settings

### ISSUE THREAD

**Issue by dawadam** [Leo4815162342/dukascopy-node#145]
Would it be possible to add a "limit" parameter in the configuration ?
Who would stop the process, once a corresponding number of candles is reached.

**Comment by Leo4815162342** [comment]
@dawadam what is your exact use case?

If we're not talking about ticks, you can always rely on start and end date for your requests. Let's say you are downloading  a day worth (from: '2023-06-18', to: '2023-06-19') of minute candles `m1`, you should always get 1440 candles.

And, for example, if you want to download 60 candles, you can request only an hour worth of candles:

```typescript
{
  timeframe: 'm1',
  dates: {
    from: '2019-01-18T00:00:00.000Z',
    to: '2019-01-18T01:00:00.000Z'
},
```

TLDR: you can infer the number of candles based on the dates (unless it's a tick data)

**Comment by dawadam** [comment]
Of course, it is possible to make calculations upstream.
Although this poses certain limits according to the assets, open or not on weekends among other things.
It would be easier to be able to limit the number of candles returned for use.
So that you don't have to worry about preparing the query in a more complex way.
